openbb_sdk/providers/alpha_vantage/openbb_alpha_vantage/models/stock_eod.py

```diff
@@ -0,0 +1,217 @@
+"""Alpha Vantage Stock End of Day fetcher."""
+
+
+from datetime import datetime
+from typing import Any, Dict, List, Literal, Optional, get_args
+
+import pandas as pd
+from dateutil.relativedelta import relativedelta
+from openbb_provider.abstract.fetcher import Fetcher
+from openbb_provider.standard_models.stock_eod import StockEODData, StockEODQueryParams
+from openbb_provider.utils.descriptions import DATA_DESCRIPTIONS
+from openbb_provider.utils.helpers import get_querystring
+from pydantic import Field, NonNegativeFloat, PositiveFloat, root_validator, validator
+
+
+class AlphaVantageStockEODQueryParams(StockEODQueryParams):
+    """Alpha Vantage Stock End of Day Query.
+
+    Source: https://www.alphavantage.co/documentation/
+    """
+
+    function: Literal[
+        "TIME_SERIES_INTRADAY",
+        "TIME_SERIES_DAILY",
+        "TIME_SERIES_WEEKLY",
+        "TIME_SERIES_MONTHLY",
+    ] = Field(
+        description="The time series of your choice. ",
+        default="TIME_SERIES_DAILY",
+    )
+    interval: Optional[Literal["1min", "5min", "15min", "30min", "60min"]] = Field(
+        description="The interval between two consecutive data points in the time series.",
+        default="60min",
+        available_on_functions=["TIME_SERIES_INTRADAY"],
+        required_on_functions=["TIME_SERIES_INTRADAY"],
+    )
+    adjusted: Optional[bool] = Field(
+        description="Output time series is adjusted by historical split and dividend events.",
+        default=True,
+        available_on_functions=["TIME_SERIES_INTRADAY"],
+    )
+    extended_hours: Optional[bool] = Field(
+        description="Extended trading hours during pre-market and after-hours.",
+        default=False,
+        available_on_functions=["TIME_SERIES_INTRADAY"],
+    )
+    month: Optional[str] = Field(
+        description="Query a specific month in history (in YYYY-MM format).",
+        default=None,
+        available_on_functions=["TIME_SERIES_INTRADAY"],
+    )
+    outputsize: Optional[Literal["compact", "full"]] = Field(
+        description="Compact returns only the latest 100 data points in the intraday "
+        "time series; full returns trailing 30 days of the most recent intraday data "
+        "if the month parameter (see above) is not specified, or the full intraday "
+        "data for a specific month in history if the month parameter is specified.",
+        default="compact",
```

Comment:
Probably want to have this set as "full" otherwise it will override any `start_date` and `end_date` settings.  Not much point in returning only the last 100 minutes (if one-minute).

Reply:
Change the default to `full`. That's it.